### PR TITLE
Add support to pass Wiql query from file in boards query

### PR DESCRIPTION
**Pull request by gauravsaralMs**
Please make sure the code is following contribution guidelines in CONTRIBUTING.md

 - [x] : This PR has a corresponding issue open in the Repository.
 - [ ] : Approach is signed off on the issue.

This is for Issue https://github.com/Azure/azure-devops-cli-extension/issues/210

@geverghe : please take a look at parameter name and help text

### CODE REVIEW

File: azure-devops/azext_devops/dev/boards/work_item.py [hunk 1]
```diff
@@ -242,10 +246,15 @@ def query_work_items(wiql=None, id=None, path=None, organization=None, project=N
         id = query.id
     if id is not None:
         query_result = client.query_by_id(id=id)
-    else:
+    if wiql is not None:
         wiql_object = Wiql()
         wiql_object.query = wiql
         query_result = client.query_by_wiql(wiql=wiql_object)
+    if wiql_file_path is not None:
```
Comment (atbagga) [inline azure-devops/azext_devops/dev/boards/work_item.py]: :Nit 

We can combine wiql and wiql file path input similar to how path and id are combined. 
```
if wiql is None and wiql_file_path is not None:
    with open(wiql_file_path, 'r') as file:
        wiql = file.read().replace('\n', '')

Then this remains as is.. 
if wiql is not None:
    wiql_object = Wiql()
    wiql_object.query = wiql
    query_result = client.query_by_wiql(wiql=wiql_object)
```

File: azure-devops/azext_devops/dev/boards/work_item.py [hunk 2]
```diff
@@ -220,18 +220,22 @@ def show_work_item(id, open=False, organization=None, detect=None):  # pylint: d
...
     """Query for a list of work items.
     :param wiql: The query in Work Item Query Language format.  Ignored if --id or --path is specified.
     :type wiql: str
+    :param wiql_file_path: The query in Work Item Query Language format in a file.
+                           Ignored if --wiql, --id or --path is specified.
+                           Please use / backslash when typing in directory path.
+    :type wiql_file_path: str
     :param id: The UUID of an existing query.  Required unless --path or --wiql are specified.
     :type id: str
     :param path: The path of an existing query.  Ignored if --id is specified.
```
Comment (geverghe) [inline azure-devops/azext_devops/dev/boards/work_item.py]: 1. What would be the priority /implementation order if both wiql and file path are specified - even though it looks to be an erroneous scenario, curious on how it is tackled?
2. Help text recommendation - "The local file path to a file containing the wiql"
3. Are there any nuances the user has to take care of when specifying path? I remember in one earlier path we were requesting for '/'. Is that the case? Further, is it just any "file" or are we expecting any particular file format - json/text?

Comment (gauravsaralMs) [reply azure-devops/azext_devops/dev/boards/work_item.py]: 1. --wiql will be used and --wiql-file-path will be ignored. This is why this line is present `Ignored if --wiql, --id or --path is specified.`
2. Sure
3. yes. Please use / backslash when typing in directory path.  It is a file containing WIQL in plane text

**Comment by gauravsaralMs** [comment]
abandon for now, as this requirement is not needed

**Pull request closed without merging**